subprojects/junit-jupiter/src/main/java/org/mockito/junit/jupiter/ConfiguredWithMockito.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.junit.jupiter;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.quality.Strictness;
+
+import java.lang.annotation.Retention;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Annotation that can configure Mockito as invoked by the {@link MockitoExtension}.
+ */
+@ExtendWith(MockitoExtension.class)
+@Retention(RUNTIME)
+public @interface ConfiguredWithMockito {
+
+    /**
+     * Configure the strictness used in this test.
+     * @return The strictness to configure, by default {@link Strictness#WARN}
+     */
+    Strictness strictness() default Strictness.WARN;
```

Comment:
New Mockito features (such as MockitoSession) uses Strictness.STRICT_STUBS by default because it helps us and the users:
 - cleaner tests that are easier to debug with STRICT_STUBS
 - it is a compatible way of introducing a new feature

How about we change the default for Jupiter to use STRICT_STUBS, too?

Reply:
I was under the impression that the WARN behavior was still default, but I just checked https://github.com/mockito/mockito/blob/45a78a03a38fd2408f0ea7a3e1342fbffcfb9237/src/main/java/org/mockito/junit/MockitoJUnitRunner.java#L152-L155 and it is indeed Strict. Will update :+1: